app/src/main/java/com/hamdan/forzenbook/data/repository/LoginRepositoryImpl.kt

```diff
@@ -4,15 +4,18 @@ import com.hamdan.forzenbook.data.database.LoginDao
 import com.hamdan.forzenbook.data.network.LoginService
 
 class LoginRepositoryImpl(
+    // TODO implement the dao FA-84
     private val dao: LoginDao,
     private val service: LoginService
 ) : LoginRepository {
     override suspend fun requestValidation(email: String): Boolean {
```

Comment:
Probably don't need a boolean here or anywhere. It feels very unstandard. It either succeeds or throws